src/dbc.cpp

```diff
@@ -123,6 +237,9 @@ void DbcParser::parse_dbc_nodes(std::istream& file_stream) {
 void DbcParser::parse_dbc_messages(const std::vector<std::string>& lines) {
 	std::smatch match;
 
+	std::vector<VALObject> sv;
+
+	VALObject obj;

```

Comment:
This come back to the comment before that the message shouldn't be parsing this. Maybe the val should be found first then passed to the signal to find the ones it needs? I guess I need to think about this some and maybe the function being a member on the signal isn't the right choice.

Reply:
The idea was first parsing the dbc file and then assigning the val object to the signal